### ISSUE TITLE
1.5.5 MacOS release actually contains the Linux executable (and so doesn't work)

### ISSUE THREAD

**Issue by JoeStrout** [Pathoschild/StardewXnbHack#14]
I just downloaded the Mac package for the 1.5.5 beta.  But after unzipping it and installing it as directed, the .command works fine at verifying the installation and copying the config files, but then when it attempts to run the actual executable, that fails with

   zsh: exec format error: ./StardewXnbHack

So I used the `file` command to see exactly what this exe is, and it says:

```$ file StardewXnbHack
StardewXnbHack: ELF 64-bit LSB pie executable, x86-64, version 1 (SYSV), dynamically linked, 
interpreter /lib64/ld-linux-x86-64.so.2, for GNU/Linux 2.6.32, 
BuildID[sha1]=1cfaaf19b37c906e12f121854b3a6b45c6c9bdf7, stripped
```

Looks to me like somebody accidentally zipped up the Linux executable into the Mac package.

**Comment by Pathoschild** [comment]
Hi! That's not a mistake, the Linux build works on macOS too. If it doesn't work for you, can you come ask in [#making-mods on the Stardew Valley Discord](https://smapi.io/community#Discord) so we can run some tests? (Feel free to ping me there with `@Pathoschild#0001`.)

**Comment by Pathoschild** [comment]
Thanks for testing! Fixed in `develop` for the upcoming 1.0.6.
